cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/common/DataprocUtils.java

```diff
@@ -229,6 +234,35 @@ public static String getSystemProjectId() {
     }
   }
 
+  /**
+   * Emit a dataproc metric.
+   **/
+  public static void emitMetric(ProvisionerContext context, String region,
+                                @Nullable Exception e, String metricName) {
```

Comment:
I would suggest swap the e and metricName. Generally we extend more arguments

Reply:
done